### PR TITLE
Add copy to clipboard button in Diagnostics page.

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: resources/js/views/Diagnostics.vue [hunk 1]
```diff
@@ -28,9 +28,35 @@ import SpaceDiagnostics from "@/components/diagnostics/SpaceDiagnostics.vue";
...
+
+const toast = useToast();
+const errorLoaded = ref<boolean>(false);
+const infoLoaded = ref<boolean>(false);
+const configurationLoaded = ref<boolean>(false);
+
+const canCopy = computed(() => errorLoaded.value && infoLoaded.value && configurationLoaded.value);
+
+function copy() {
+	const errors = document.getElementById("ErrorsData");
```
Comment (tinohager) [inline resources/js/views/Diagnostics.vue]: Whoa, that's definitely not the way to do it! 💩 

it is possible to transmit the data to the outside via the emit
https://vuejs.org/guide/typescript/composition-api#typing-component-emits

Comment (ildyria) [reply resources/js/views/Diagnostics.vue]: I know, but I didn't feel like it. :laughing: 
And that was simpler to "copy" the content that way.

Comment (tinohager) [reply resources/js/views/Diagnostics.vue]: That's still not the way to do it. The other is more standardized 😄

File: resources/js/views/Diagnostics.vue [hunk 2]
```diff
@@ -28,9 +28,35 @@ import SpaceDiagnostics from "@/components/diagnostics/SpaceDiagnostics.vue";
...
+	const info = document.getElementById("InfoData");
+	const Configuration = document.getElementById("ConfigurationData");
+	const errorLines = errors?.innerText.split("\n") ?? [];
+	let errorText = "";
+	for (let i = 0; i < errorLines.length; i++) {
+		errorText += `${errorLines[i].padEnd(7)}: ${errorLines[i + 1]}\n`;
+		i += 1;
```
Comment (d7415) [inline resources/js/views/Diagnostics.vue]:
```suggestion
	for (let i = 0; i < errorLines.length; i+=2) {
		errorText += `${errorLines[i].padEnd(7)}: ${errorLines[i + 1]}\n`;
```
?

**Review by d7415 — APPROVED** [review]

**Pull request merged**